paper/paper.Rmd

```diff
@@ -328,12 +332,12 @@ Implementations of blocking methods often provide additional options to fine tun
 
 #### Spatial blocking --- blockCV
 
-The "Spatial blocking" from package \pkg{blockCV} comes with various options to customize the zone creation.
-Besides the choice of relying on an initial random  (default) (`selection = "random"`), systematic (`selection = "systematic"`) or checkerboard pattern (`selection = "checkerboard"`) for the final partition aggregation, support for SDM specific background data is built-in.
-Supplying a numeric value in meters via argument `theRange` (named just `range` in {mlr3spatiotempcv}) will create a quadratic block pattern across the study area.
-Ultimately, users can also supply a user-defined polygon which defines the zones.
+The "Spatial blocking" from package \pkg{blockCV} comes with various blocking options such as random (default) (`selection = "random"`), systematic (`selection = "systematic"`) or checkerboard pattern (`selection = "checkerboard"`).
+Additionally, the user might supply a numeric value in meters via argument `theRange` (named just `range` in {mlr3spatiotempcv}) which will ensure a quadratic block pattern across the study area.
+<!--JM: to stay consistent please add the argument which lets the user indicate a user-defined polygon or are you referring to the rectangular block design specified with rows and cols? If so, say so. If not, clarify and additionally mention the rectangular block design here-->
```

Comment:
If you don't think that `rasterLayer` is useful to the user, omit it! If you mention it, pls also mention argument `rasterLayer` since you offer the arguments for all other options.

Reply:
The underlying issue here is that I cannot/don't want to talk about every single parameter of the underlying method (in detail).
To me discussing the `selection` argument was more important than showcasing that one could also pass a custom polygon and then this one is used for the blocks.

I now mentioned the argument name in the sentence like

> Ultimately, users can also supply a user-defined polygon via argument `rasterLayer` to define the zones.

Do you think that's ok?

Reply:
I think I would emphasize the advantages of the mlr3 ecosystem in section **Should new methods be added to existing ML ecosystems or made available in a new package?**, rename the section accordingly, and also shortly refer to all the additional nice things this ecosystem offers. And in this section you could also name a few disadvantages.

Reply:
Now I need to go to sleep -:), the best of luck with your submission, JSS was at least famous for its overly long publication times (see [here](https://www.r-bloggers.com/2014/06/time-to-accept-it-publishing-in-the-journal-of-statistical-software/)) but maybe that has changed in the meantime.

Reply:
Thanks! I renamed the section and rephrased a bit.

I am aware of the long review time in JSS but (luckily) this is not a blocker for the overarching plans :)